src/usr/local/www/widgets/widgets/openvpn.widget.php

```diff
@@ -46,7 +46,12 @@ function printPanel($widgetkey) {
 			$got_ovpn_server = true;
 
 		$opstring .= "<div class=\"widget panel panel-default\">";
-		$opstring .=	"<div class=\"panel-heading\"><h2 class=\"panel-title\">" . htmlspecialchars($server['name']) . "</h2></div>";
+		if ($server['conns'][0]['common_name'] != '[error]') {
+			$opstring .=	"<div class=\"panel-heading\"><h2 class=\"panel-title\">" . htmlspecialchars($server['name']) . " " . gettext('Client Connections') . ": " . sizeof($server['conns']) . "</h2></div>";
+		}
+		else {
+			$opstring .=	"<div class=\"panel-heading\"><h2 class=\"panel-title\">" . htmlspecialchars($server['name']) . "</h2></div>";
```

Comment:
You are assigning a good portion of the same string to `$optstring` twice. It can be improved as:
```php
$opstring .= "<div class=\"panel-heading\"><h2 class=\"panel-title\">" .
    htmlspecialchars($server['name']);
if ($server['conns'][0]['common_name'] != '[error]') {
	$optstring .= " " . gettext('Client Connections') . ": " .
	    sizeof($server['conns']);
}
$optstring .= "</h2></div>";
```